### PR TITLE
Update support matrix for 7.5

### PR DESCRIPTION
**Pull request by amottier**

### CODE REVIEW

**Comment by educhastenier** [comment]
retest this please

File: md/hardware-and-software-requirements.md [hunk 1]
```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 ## Software
 
 Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Portal).
-| | 7.0.x <br/> 7.1.x | 7.2.x <br/> 7.3.x  | 7.4.x | 7.5.x
```
Comment (educhastenier) [inline md/hardware-and-software-requirements.md:20]: Is this new presentation mode validated?
This is a drastic presentation change...

File: md/hardware-and-software-requirements.md [hunk 2]
```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
...
-| iOS 7.1 -- Safari -- Approved on Apple iPhone 5 | ✔ | ✔ | ✔ | ✔ |
-| Android 4.4 -- Chrome -- Approved on Nexus 5 | ✔ | ✔ | ✔ | ✔ |
+| Mozilla Firefox | latest version |
+| Google Chrome | latest version |
+| Apple Safari | latest version |
 
 Notes:
 
 1. MySQL must be configured with innoDB storage engine.
-2. **Warning (7.0.x only, fixed starting 7.1.0) :** There are known issues with the management of XA transactions by MySQL engine and driver: see MySQL bugs [17343](http://bugs.mysql.com/bug.php?id=17343) and [12161](http://bugs.mysql.com/bug.php?id=12161) for more details.
```
Comment (educhastenier) [inline md/hardware-and-software-requirements.md:55]: Why removing this part if MySQL version is not updated?

File: md/hardware-and-software-requirements.md [hunk 3]
```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
...
+2. There is a known issue between Bitronix (the transaction manager shipped by Bonitasoft for the Tomcat bundle and inside the Deploy bundle for Tomcat) and Microsoft SQL Server JDBC driver
 (refer to: [MSDN note](https://msdn.microsoft.com/en-us/library/aa342335.aspx), [Bitronix note](http://bitronix-transaction-manager.10986.n7.nabble.com/Failed-to-recover-SQL-Server-Restart-td148.html)).
-Therefore, using Bitronix as a Transaction Manager with SQL Server does not work and is not supported. To use SQL Server database requires that you use the JBoss or WildFly bundle provided by Bonitasoft.
-4. Bonita BPM can be executed on Java 8 and above. All development artifacts (connectors, REST API extensions, etc) must be compiled with Java 8 byte code (target version).
+Therefore, using Bitronix as a transaction manager with SQL Server does not work and is not supported. To use SQL Server database requires that you use the WildFly bundle provided by Bonitasoft.
+3. Bonita BPM can be executed on Java 8 and above. All development artifacts (connectors, REST API extensions, etc) must be compiled with Java 8 byte code (target version).
```
Comment (educhastenier) [inline md/hardware-and-software-requirements.md:52]: No need to re-number all bullet points, as soon as they are numbers.
This avoids false-positive potential conflicts when merging.

File: md/hardware-and-software-requirements.md (hunk 2 above)
Comment (amottier) [reply md/hardware-and-software-requirements.md:55]: Warning only apply to version 7.0.x. As page focus on 7.4.x removing the warning seems ok to me.

File: md/hardware-and-software-requirements.md (hunk 1 above)
Comment (amottier) [reply md/hardware-and-software-requirements.md:20]: Yes modification is validated by Philippe, Haris and myself.

File: md/hardware-and-software-requirements.md [hunk 4]
```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
...
-| Oracle Java SE Runtime Environment 7 u67 | ✔ | ✔ | ✔ | |
-| Oracle Java SE Runtime Environment 8 u40 (see note 4) | ✔ | ✔ |  ✔ | ✔ |
-| OpenJDK 7u65 | ✔ | ✔ | ✔ | |
-| OpenJDK 8 | | | | ✔ |
+| Microsoft Windows Server | 2012 R2 64 bits and higher |
+| Red Hat Enterprise Linux |  6.5 64 bits and higher |
+| Ubuntu | 14.04 LTS 64 bits and higher |
+| **Java Virtual Machine** |
+| Oracle Java SE Runtime Environment | 8u112 (see note 3) |
+| OpenJDK | 8u112 (see note 3) |
```
Comment (educhastenier) [inline md/hardware-and-software-requirements.md:28]: Are you sure version 8u112 is valid for Oracle JDK and OpenJDK?

Comment (amottier) [reply md/hardware-and-software-requirements.md:28]: Yes.
Oracle: http://www.oracle.com/technetwork/java/javase/8u112-relnotes-3124973.html
OpenJDK: http://openjdk.java.net/projects/jdk8u/releases/8u112.html

File: md/hardware-and-software-requirements.md (hunk 2 above)
Comment (educhastenier) [reply md/hardware-and-software-requirements.md:55]: 7.5.x, you mean

**Comment by educhastenier** [comment]
Looks good to me

File: md/hardware-and-software-requirements.md (hunk 2 above)
Comment (amottier) [reply md/hardware-and-software-requirements.md:55]: Yes sorry :)

**Pull request merged**